src/java/picard/illumina/IlluminaBasecallsToFastq.java

```diff
@@ -68,15 +68,45 @@
-                "of the barcode read, starting with 1.",
-        usageShort = "Generate fastq file(s) from data in an Illumina basecalls output directory",
-        programGroup = Illumina.class
+  usage = IlluminaBasecallsToFastq.USAGE_SUMMARY + IlluminaBasecallsToFastq.USAGE_DETAILS,
+  usageShort = IlluminaBasecallsToFastq.USAGE_SUMMARY,
+  programGroup = Illumina.class
 )
 public class IlluminaBasecallsToFastq extends CommandLineProgram {
+  static final String USAGE_SUMMARY = "Generate fastq file(s) from Illumina basecall read data.  ";
+  static final String USAGE_DETAILS = "<p>This tool generates FASTQ files from data in an Illumina BaseCalls output directory.  " +
+          "Separate FASTQ files are created for each template, barcode, and index (molecular barcode) read.  Briefly, the template reads are the target sequence of your experiment, the barcode sequence reads facilitate sample demultiplexing, and the index reads help mitigate instrument phasing errors.  For additional information on the read types, please see the following reference <a href'=http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3245947/'>here</a>.</p>" +
+          "<p>In the absence of sample pooling (multiplexing) and/or barcodes, then an OUTPUT_PREFIX (file directory) must be provided as the sample identifier.  For multiplexed samples, a MULTIPLEX_PARAMS file must be specified.  The MULTIPLEX_PARAMS file contains the list of sample barcodes used to sort template, barcode, and index reads.  It is essentially the same as the BARCODE_FILE used in the" +
+          "<a href='http://broadinstitute.github.io/picard/command-line-overview.html#ExtractIlluminaBarcodes'>ExtractIlluminaBarcodes</a> tool.     "+
+          "" +
+          "<p>Files from this tool use the following naming format: {prefix}.{type}_{number}.fastq with the {prefix} indicating the sample barcode, " +
+          "the {type} indicating the types of reads e.g. index, barcode, or blank (if it contains a template read).  The {number} indicates the" +
+          " read number, either first (1) or second (2) for paired-end sequencing.  " +
+
+"<h4>Usage examples:</h4>" +
+"<pre>" +
+"Example 1: Sample(s) with either no barcode or barcoded without multiplexing <br />" +
+"java -jar picard.jar IlluminaBasecallsToFastq \\<br />"+
+"      READ_STRUCTURE=25T8B25T \\<br />"+
+"      BASECALLS_DIR=s_1_1101.bcl \\<br />"+
+"      LANE=001 \\<br />"+
+"      OUTPUT_PREFIX=noBarcode.1 \\<br />"+
+"      RUN_BARCODE=run15 \\<br />"+
+"      FLOWCELL_BARCODE=abcdeACXX* <br /><br />" +
+
+"Example 2: Multiplexed samples <br />" +
+"java -jar picard.jar IlluminaBasecallsToFastq \\<br />"+
+"      READ_STRUCTURE=25T8B25T \\<br />"+
+"      BASECALLS_DIR=s_1_1101.bcl \\<br />"+
+"      LANE=001 \\<br />"+
+"      MULTIPLEX_PARAMS=demultiplexed_output.txt \\<br />"+
+"      RUN_BARCODE=run15 \\<br />"+
+"      FLOWCELL_BARCODE=abcdeACXX* <br />" +
```

Comment:
Don't use an asterisk in a command to refer to a footnote! People will think it's part of the command.